apps/api/.env

```diff
@@ -1,3 +1,3 @@
 ACCESS_TOKEN_SECRET=[YOUR_ACCESS_TOKEN_SECRET_HERE]
 REFRESH_TOKEN_SECRET=[YOUR_REFRESH_TOKEN_SECRET_HERE]
-MONGO_URL=[YOUR_MONGO_CONNECTION_STRING_HERE]
\ No newline at end of file
+MONGO_URL=mongodb+srv://anna:AxIiq4HfpFxNTnbL@cluster0.gap9e.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0
```

Comment:
We need to remove this from the PR since has sensitive information

Reply:
Done